kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -197,13 +197,13 @@ namespace Kratos
 
 	template<std::size_t TDim>
 	unsigned int CalculateDiscontinuousDistanceToSkinProcess<TDim>::ComputeEdgesIntersections(
-		Element& rElement1, 
+		Element& rElement1,
 		const PointerVector<GeometricalObject>& rIntersectedObjects,
 		std::vector<unsigned int> &rCutEdgesVector,
       	std::vector<array_1d <double,3> > &rIntersectionPointsArray)
 	{
 		auto &r_geometry = rElement1.GetGeometry();
-		const auto r_edges_container = r_geometry.Edges();
+		const auto r_edges_container = r_geometry.GenerateEdges();
```

Comment:
@loumalouomega as i recall, in order to get the same behaviour as before we should call GenerateBoundaryEntities, am I wrong? if that is the case than the warning on Edges is wrongly pointing to GenerateEdges, which will change the behaviour

Reply:
This is just to remove the Warnings, for now I would only change the function. I would replace with the new one in a future iteration, otherwise you can broke current behaviour